algorithms/list/list.py

```diff
@@ -0,0 +1,79 @@
+"""
+Python arrays are known as 'lists'
+"""
```

Comment:
This is just showing usage of Python lists. I don't think it fits in this collection.